### DISCUSSION TITLE
Install in GPO (Active Directory)

### DISCUSSION THREAD

**Discussion by ki4arl** [valinet/ExplorerPatcher#3858] [Q&A]
Good day to all, I want to install this program through Active Directory, using GPO, I wrote a fairly minimal script via bat

``` bat
powershell -Command "Add-MpPreference -ExclusionPath '\\local\sysdc\ExplorerPatcher\ep_setup.exe'"
powershell -Command "Add-MpPreference -Exclusion Path 'C:\Program Files\ExplorerPatcher'"
powershell -Command "Add-Preference -ExclusionPath '$env:APPDATA\ExplorerPatcher'"
powershell -Command "Add-MpPreference -ExclusionPath 'C:\Windows\dxgi.dll'"
powershell -Command "Add-MpPreference -ExclusionPath 'C:\Windows\SystemApps\Microsoft.Windows.StartMenuExperienceHost_cw5n1h2txyew'"
powershell -Command "Add-MpPreference -ExclusionPath 'C:\Windows\SystemApps\ShellExperienceHost_cw5n1h2txyew'"
\\local\sysdc\ExplorerPatcher\ep_setup.exe
```

It is installed, but the start and search menu does not work, I do not even know what to do, please advise

Version 22631.3737

**Comment by pyrates999** [comment]
Please post complete version of EP.

And those commands require admin privileges to run and to also update EP.  They must be run after the user has logged in as well as one of them references one of the users environmental variables.

**Reply by ki4arl** [reply]
22621.3880.66.6 version of EP

I'm throwing the team in 
Computer configuration - Windows configuration - Scripts (startup/shutdown) 
I put this bat on startup, this program is installed, everything appears, but the start menu and search does not work yet unless you manually install this program

**Reply by pyrates999** [reply]
Check if the windows defender exclusions were added.

**Reply by ki4arl** [reply]
I checked, unfortunately for some reason it is not being added

**Reply by pyrates999** [reply]
You need to run this with admin privileges under each user.  So each user must be a member of the local group administrators, in order for this to work.

**Reply by ki4arl** [reply]
Is there any other way to do it? I want it to be a computer, because I don't want to grant administrator rights to every account

**Reply by pyrates999** [reply]
No, because you need administrative privileges to add the exclusions to windows defender.
